native/ios/Integreat/Info.plist

```diff
@@ -66,6 +66,41 @@
 	<string>Integreat wants to use your location to find nearby cities.</string>
 	<key>NSLocationAlwaysAndWhenInUseUsageDescription</key>
 	<string>Integreat wants to use your location to find nearby cities.</string>
+    <key>NSPrivacyAccessedAPITypes</key>
+        <array>
+            <dict>
+                <key>NSPrivacyAccessedAPIType</key>
+                <string>NSPrivacyAccessedAPICategoryDiskSpace</string>
+                <key>NSPrivacyAccessedAPITypeReasons</key>
+                <array>
+                <string>E174.1</string>
+                </array>
+            </dict>
+            <dict>
+                <key>NSPrivacyAccessedAPIType</key>
+                <string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+                <key>NSPrivacyAccessedAPITypeReasons</key>
+                <array>
+                <string>3B52.1</string>
+                </array>
```

Comment:
❓ Why do we actually access timestamps? Is that for the cache?

Reply:
no idea as i mentioned. i guess its for offline data

Reply:
I think this should be `C617.1`, i,e
> Declare this reason to access the timestamps, size, or other metadata of files inside the app container, app group container, or the app’s CloudKit container.

If it is about the offline data (which I strongly assume as well) it refers to files inside the app container and NOT files the user granted access to (which we don't have at all in Integreat).